pyproject.toml

```diff
@@ -116,4 +116,5 @@ ban-relative-imports = "all"
 "onnx/reference/ops/aionnx_preview_training/__init__.py" = ["F401"]
 "onnx/reference/ops/aionnx_preview_training/_op_list.py" = ["F401"]
 "onnx/reference/ops/experimental/__init__.py" = ["F401"]
+"onnx/test/reference_evaluator_test.py": ["C408"]
```

Comment:
I would add a comment on why we need to disable this. I think C408 can potentially be disabled all together if we see the use of dict() for creating kwargs.